alibaba-dragonwell/issues.md

```diff
@@ -0,0 +1 @@
+[GitHub](%%GITHUB-REPO%%/issues);
```

Comment:
This is very similar to the default file, so you could delete `issues.md` entirely and have one less file to think about (unless you're really keen on it saying `GitHub` instead of `https://github.com/dragonwell-releng/dragonwell-containers/issues` in the link text).

Reply:
removed issues.md